c/src/ml-api-service-extension.c

```diff
@@ -1236,13 +1332,54 @@ ml_service_get_information (ml_service_h handle, const char *name, char **value)
     }
     default:
       _ml_error_report_return (ML_ERROR_NOT_SUPPORTED,
-          "The ml-service type '%d' does not support this function.", mls->type);
+          "The ml-service type '%d' does not support this function.",
+          mls->type);
       break;
   }
 
   return status;
 }
 
+/**
+ * @brief Sets the information in ml-service handle.
+ */
+int
+ml_service_set_information (ml_service_h handle, const char *name, void *value,
```

Comment:
Discussion: set/get string-type information 
ml_service_set_information (ml_service_h handle, const char *name, const char *value);

Reply:
Changed from `void *` to `char *`. Thanks!